### PR TITLE
Added html for section 5

### PR DESCRIPTION
**Pull request by Galin7777**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,36 @@ <h2 class="cashback__title">and cashback!</h2>
       </div>
     </section>
 
-    <section>5</section> 
+    <section class="clients" id="clients">
+      <div class="clients__wrapper">
+        <ul class="clients__brands"> 
+          <li class="clients__brand">
+            <img src="./assets/brands/brand1.svg" alt="Brand 1">
```
Comment (Palych18) [inline index.html:79]: Stick to one style

**Review by Palych18 — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Galin7777) [reply index.html]: Fixed

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**